util/MsSql/backup-db.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/sh
 
-# Backup timestamp
-export now=${1:-$(date +%Y%m%d_%H%M%S)}
+while true
+do
+  # Sleep until next day
+  [ "$1" == "loop" ] && sleep $((24 * 3600 - (`date +%H` * 3600 + `date +%M` * 60 + `date +%S`)))
 
-# Do a new backup
-/opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P ${SA_PASSWORD} -i /backup-db.sql
+  # Backup timestamp
+  export now=$(date +%Y%m%d_%H%M%S)
```

Comment:
`${1:-` was a good add:

> One could do things like this (I do actually) :
`docker exec -ti d323a1c57728 /backup-db.sh $(date +%Y%m%d_%H%M%S)`
(in a more global backup plan for example)

https://github.com/bitwarden/server/pull/298/files#r191313662

Any reason why it was removed? Seems accidental to me.

Thanks

Reply:
@Mart124

Reply:
HI @haneef95, it has been removed now that you can set the container timezone properly, your backups will then be properly timestamped :
```
# cat docker-compose.override.yml 
version: '3'

services:
  mssql:
    environment:
      - TZ=Europe/Berlin
      - BACKUP_DB=0
```
Last line is an example to show you that you can also disable the built-in backup schedule itself.

Does it suit your needs ?

Reply:
Thanks @Mart124, I see. And I also later realised that when executed from `entrypoint.sh` it'll likely name the file as `loop`:
https://github.com/bitwarden/server/blob/9f96e4ce90a24f9830bf1ab117ce3d3167d460a7/util/MsSql/entrypoint.sh#L62

So, I guess it had to be removed.

I've submitted a PR #1720 now however, with some additional features, including the ability to rename the backup file and/or give it a suffix. These should suit some of my needs and of course the community. Feel free to drop a line on that with your thoughts. 😄